zerver/views/users.py

```diff
@@ -162,9 +166,16 @@ def _deactivate_user_profile_backend(
     user_profile: UserProfile,
     target: UserProfile,
     *,
+    is_spammer: Json[bool] = False,
+    message_delete_action: Json[list[int]] | None = None,
```

Comment:
I am not sure of this field name now that we pass a list of integers. Would be good to discuss this on #api-design.